examples/use-async-data/app.vue

```diff
@@ -3,7 +3,7 @@ const { data, refresh, pending } = await useAsyncData('/api/hello', () => $fetch
 </script>
 
 <template>
-  <NuxtExampleLayout example="use-async-data" show-tips="true">
+  <NuxtExampleLayout example="use-async-data" :show-tips="true">
```

Comment:
Great catch. Or perhaps for simplicity's sake?

```suggestion
  <NuxtExampleLayout example="use-async-data" show-tips>
```